cmd/rook/ceph/osd.go

```diff
@@ -101,6 +102,7 @@ func addOSDFlags(command *cobra.Command) {
 	// flags for removing OSDs that are unhealthy or otherwise should be purged from the cluster
 	osdRemoveCmd.Flags().StringVar(&osdIDsToRemove, "osd-ids", "", "OSD IDs to remove from the cluster")
 	osdRemoveCmd.Flags().BoolVar(&preservePVC, "preserve-pvc", false, "Whether PVCs for OSDs will be deleted")
+	osdRemoveCmd.Flags().BoolVar(&forceOSDRemoval, "force-osd-removal", false, "Whether to force remove the OSD")
```

Comment:
How about if we add the default `--force-osd-removal=false` in the sample osd-purge.yaml? Then it is more obvious as an option for users.